starport/pkg/cosmosanalysis/module/module_test.go

```diff
@@ -2,11 +2,27 @@ package module
 
 import (
 	"context"
+	"os"
+	"path/filepath"
 	"testing"
 
 	"github.com/kr/pretty"
 )
 
 func TestExampleDiscover(t *testing.T) {
-	pretty.Println(Discover(context.Background(), "/home/ilker/Documents/code/src/github.com/tendermint/starport/local_test/mars"))
+	goPath := os.Getenv("GOPATH")
+	t.Run("test folder without proto files", func(t *testing.T) {
+		pretty.Println(Discover(
+			context.Background(),
+			filepath.Join(goPath, "src/github.com/tendermint/starport/local_test/mars"),
```

Comment:
Can we scaffold a chain, tarball it and put under `testdata` dir inside `cosmosanalysis/module`? then we can extract it in this test and actually do some assertions.